client/src/commons/Toggle/Toggle.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { ToggleButtonGroup, ToggleButton, ToggleButtonGroupProps } from '@material-ui/lab';
+
+import { useStyles } from './ToggleStyles';
+import theme from 'styles/theme';
+
+const Toggle: React.FC<Props> = (props: Props): JSX.Element => {
+    const classes = useStyles({});
+
+    const { firstOption, secondOption, value, ...rest } = props;
+    const activeButtonStyle = {
+        backgroundColor: theme.palette.primary.main, 
+        color: 'white'
+    }
+
+    return (
+        <ToggleButtonGroup value={value} exclusive {...rest}>
+            <ToggleButton className={classes.toggle} 
+                style={!value ? activeButtonStyle : {}}
```

Comment:
use & instead

Reply:
That  impossible since it translate it to false and ts doent allow it